sn_client/src/audit/spend_dag_building.rs

```diff
@@ -200,4 +204,15 @@ impl Client {
         info!("Verified all the way to known spends or genesis! Through {depth} generations, verifying {n} transactions in {elapsed:?}");
         Ok(())
     }
+
+    /// Extends an existing SpendDag starting from the utxos in this DAG
+    /// Covers the entirety of currently existing Spends if the DAG was built from Genesis
+    pub async fn spend_dag_continue_from_utxos(&self, dag: &mut SpendDag) -> WalletResult<()> {
```

Comment:
hmm...  what's the different between this to be above `spend_dag_extend` function ?
`continue_from_utxos` is also an `extend` ?

maybe the `spend_dag_extend` shall be renamed to `dag_from_spend_address` ?

Reply:
Yeah I agree, basically `spend_dag_continue_from_utxos` extends the DAG in all directions from the current UTXOs at the extremities. That's total collection, the point of this is to collect all Spends that exist on the Network. 

`spend_dag_extend` is partial extension from one spend back to either Genesis or a known Spend that we already have in our DAG. It is adding one Spend and all its ancestors to our current DAG. 

So both are extension, but one is only in one direction and the other one is in all directions. 

`dag_from_spend_address` makes me think that we build a dag from an address, maybe we need to add an extend somewhere. 
How is `spend_dag_extend_to_spend_address`?